pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup

```diff
@@ -111,3 +111,11 @@ for srv in $EVESRVICES; do
         /bin/echo "0-$eve_cgroup_cpus_limit" > /sys/fs/cgroup/cpuset/eve/services/"${srv}"/cpuset.cpus
     fi
 done
+
+#Creating cgroup for eve-user-apps
+mkdir -p /sys/fs/cgroup/memory/eve-user-apps
```

Comment:
Why is this needed if all eve-user-apps are always started by containerd and we have constraints enforced from that side?

Reply:
You mean constraints per app?
We have no limit for the set of all apps, so we faced with OOM killer for zedbox, not for apps.

Reply:
I don't understand -- we have limits for each app. The only issue I could see if those don't add up to the # you want them to add up to. That's actually a much bigger issue with resource enforcement (currently controllers don't track wether you have enough resources for everything you specify in the config).

IOW, what you're doing here could be done much simpler if EVE start checking that all resources add up to the total resources available on the system.